rdrf/report/schema.py

```diff
@@ -395,10 +396,11 @@ def consent_section_resolver(parent, _info):
 
     consent_fields = {}
     for consent_section in registry.consent_sections.all():
-        consent_fields[consent_section.code] = graphene.Field(type(f"DynamicConsentSection_{consent_section.registry.code}_{consent_section.code}",
-                                                              (graphene.ObjectType,),
-                                                              get_consent_question_fields(consent_section)))
-        consent_fields[f'resolve_{consent_section.code}'] = consent_section_resolver
```

Comment:
When I run a report containing consents, it still generates the query with number prefixes and fails:

```graphql
query AllPatientsQuery($filterArgs: PatientFilterType) { mnd { allPatients(filterArgs: $filterArgs) { patients(sort: ["id"], offset: 1, limit: 1) { id familyName givenNames consents { 1ReadPDF { IhaveRead { answer firstSave lastUpdate } IUnderstand { answer firstSave lastUpdate } AgreeAskedQuestions { answer firstSave lastUpdate } } 2PatientRegConsent { ClinicalRegConsent { answer firstSave lastUpdate } AgreeAccessHealth { answer firstSave lastUpdate } } 3Linking { clinicalPatientLink { answer firstSave lastUpdate } UnderstandLink { answer firstSave lastUpdate } emegContactAgree { answer firstSave lastUpdate } AMNDRLink { answer firstSave lastUpdate } SALSALink { answer firstSave lastUpdate } } 5DataStoreandShare { StoreAgree { answer firstSave lastUpdate } futureResearch { answer firstSave lastUpdate } contactAgree { answer firstSave lastUpdate } docuStore { answer firstSave lastUpdate } releaseInfo { answer firstSave lastUpdate } } } } } } }
```